components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/internal/ServiceReferenceHolder.java

```diff
@@ -50,16 +49,16 @@ public ConfigProvider getConfigProvider() {
     }
 
     public APIMConfigurations getAPIMConfiguration() {
-        /*try {
+        try {
             config = ServiceReferenceHolder.getInstance().getConfigProvider()
                     .getConfigurationObject(APIMConfigurations.class);
         } catch (CarbonConfigurationException e) {
-            log.error("error getting config", e);
-        }*/
+            log.error("error getting config : org.wso2.carbon.apimgt.core.internal.APIMConfiguration", e);
+        }
 
         if (config == null) {
             config = new APIMConfigurations();
-            log.info("Setting default configurations");
+            log.info("org.wso2.carbon.apimgt.core.internal.ServiceReferenceHolder : Setting default configurations");
```

Comment:
Do we need to manually add FQN in log message? I think this is configurable using log4j pattern configuration